lib/reporters/cli/index.js

```diff
@@ -70,6 +71,8 @@ extractSNR = function (executions) {
  * @param {Boolean=} reporterOptions.noFailures - Boolean flag to turn off failure reporting altogether, if set to true.
  * @param {Boolean=} reporterOptions.noConsole - Boolean flag to turn off console logging, if set to true.
  * @param {Boolean=} reporterOptions.noBanner - Boolean flag to turn off newman banner, if set to true.
+ * @param {String=} reporterOptions.verboseOutputLimit - Specify the truncation size of body in verbose CLI output.
```

Comment:
We do not need to introduce two parameters for this. In the grand scheme of things this is a small feature and one parameter should do. Here's what we can do:

1. We ALWAYS clip. No matter what.
2. Providing `Infinity` as an input option can then automatically prevent clipping with least complicated codes. (And we mention this behaviour in docs as a short byline.)
3. We take only one option `--reporter-cli-truncate-body-output` (people will 99% chance refer documentation to get to this, hence we name accordingly.)

Check for isNaN much earlier and prevent NaN from flowing too much down the code flow.